packages/key-management/src/types.ts

```diff
@@ -93,6 +97,7 @@ export interface SerializableKeyAgentDataBase {
   chainId: Cardano.ChainId;
   accountIndex: number;
   extendedAccountPublicKey: Crypto.Bip32PublicKeyHex;
+  purpose: KeyPurpose;
```

Comment:
This should be optional for backwards-compatibility, as there may be already key agent data serialized/stored
```suggestion
  purpose?: KeyPurpose;
```

Reply:
Fixed on 1ab63d5